sdk/inc/azure/iot/az_iot_hub_client.h

```diff
@@ -183,21 +184,21 @@ AZ_NODISCARD az_result az_iot_hub_client_sas_get_signature(
  * @param[in] client The #az_iot_hub_client to use for this call.
  * @param[in] base64_hmac_sha256_signature The Base64 encoded value of the HMAC-SHA256(signature,
  *                                         SharedAccessKey). The signature is obtained by using
- *                                         #az_iot_hub_client_sas_get_signature.
+ *                                         az_iot_hub_client_sas_get_signature().
  * @param[in] token_expiration_epoch_time The time, in seconds, from 1/1/1970.
  *                                        It MUST be the same value passed to
- *                                        az_iot_hub_client_sas_get_signature.
+ *                                        az_iot_hub_client_sas_get_signature().
  * @param[in] key_name The Shared Access Key Name (Policy Name). This is optional. For security
  *                     reasons we recommend using one key per device instead of using a global
  *                     policy key.
  * @param[out] mqtt_password A char buffer with sufficient capacity to hold the MQTT password.
  * @param[in] mqtt_password_size The size, in bytes of \p mqtt_password.
  * @param[out] out_mqtt_password_length __[nullable]__ Contains the string length, in bytes, of
  *                                                     \p mqtt_password. Can be `NULL`.
- * @return #az_result.
- *         #AZ_OK if successful. In this case, `mqtt_password` will contain a null-terminated string
- *           with the password that needs to be passed to the MQTT client.
- *         #AZ_ERROR_INSUFFICIENT_SPAN_SIZE If `mqtt_password` does not have enough size.
+ * @return An #az_result value indicating the result of the operation.
+ * @retval #AZ_OK If the operation was successful. In this case, \p mqtt_password will contain a

```

Comment:
For `retval` don't use sentences starting with condition, like if, but rather plainly state the cause of that error code. This is because doxygen generates them in tabular format.

```suggestion
 * @retval #AZ_OK The operation was successful. In this case, \p mqtt_password will contain a

```